packages/web-sockets/.eslintrc.json

```
@@ -0,0 +1,19 @@
+{
+  "env": {
+    "commonjs": true,
+    "es2021": true,
+    "node": true
+  },
+  "extends": [
+    "airbnb-base"
+  ],
+  "parserOptions": {
+    "ecmaVersion": "latest"
+  },
+  "plugins": ["import"],
+  "settings": {
+  	"import/core-modules": [ "dotenv" ]
```

Can you replace the `tab` here for spaces?
```suggestion
    "import/core-modules": [ "dotenv" ]
```

---

But is not a tab, settings is dictionary that's why I added two more spaces